### PR TITLE
Update colinmollenhour/php-redis-session-abstract

### PR DESCRIPTION
**Pull request by pmclain**
### Description
Update colinmollenhour/php-redis-session-abstract to version supporting PHP 7.2.

### Fixed Issues (if relevant)
1. magento-engcom/php-7.2-support#1

### Manual testing scenarios
1. Install Magento
2. Configure caches to use Redis

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
@pmclain thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php [hunk 1]
```diff
@@ -115,6 +135,11 @@ class Config implements \Cm\RedisSession\Handler\ConfigInterface
      */
     const SESSION_MAX_LIFETIME = 31536000;
 
+    /**
+     * Try to break lock for at most this many seconds
+     */
+    const DEFAULT_FAIL_AFTER = 15;
```
Comment (buskamuza) [inline lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php:141]: Why is this one not configurable? Comparing to `PARAM_BREAK_AFTER`?

Comment (buskamuza) [reply lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php:141]: It's also necessary to update `setup:config:set` with new params (in `\Magento\Setup\Model\ConfigOptionsList\Session`).

Could you please also update http://devdocs.magento.com/guides/v2.2/config-guide/redis/redis-session.html ?

**Review by buskamuza — CHANGES_REQUESTED** [review]

File: lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php (hunk 1 above)
Comment (pmclain) [reply lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php:141]: I copied `DEFAULT_FAIL_AFTER` from 2.2-develop https://github.com/magento/magento2/blob/0839bd421bd5812e9aa7c943b89ecae9d5008832/lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php#L121 Should I change it to be configurable here?

Thanks for pointing out the changes required to `setup:config:set`. I'll get that taken care of.

Comment (buskamuza) [reply lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php:141]: Got it... ok, let's leave it as is.

**Review by buskamuza — APPROVED** [review]

**Pull request merged**